apps/web/screens/queries/IcelandicGovernmentInstitutionVacancies.ts

```diff
@@ -15,6 +15,8 @@ export const GET_ICELANDIC_GOVERNMENT_INSTITUTION_VACANCIES = gql`
         applicationDeadlineTo
         institutionName
         logoUrl
+        creationDate
```

Comment:
You are only displaying these values in case the DEBUG variable is set to true right?

What's the game plan, are these values something that we want to show to the user? Or just debug values that you temporarily added to double check.

I suggest we skip this and remove the DEBUG variable